src/Record.js

```diff
@@ -28,11 +28,34 @@ import { asImmutable } from './methods/asImmutable';
 
 import invariant from './utils/invariant';
 import quoteString from './utils/quoteString';
+import { isImmutable } from './predicates/isImmutable';
+
+function convertDefaultValues(defaultValues) {
+  if (isRecord(defaultValues)) {
+    return defaultValues.toObject();
+  }
+
+  if (isImmutable(defaultValues)) {
+    throw new Error(
+      'Can not call `Record` an immutable Collection as default values. You mant want to use a plain javascript object instead.'
```

Comment:
You wrote "mant", but I think you can just say: "Use a plain javascript object instead."